plugins/woocommerce/tests/legacy/unit-tests/woocommerce-admin/api/admin-notes.php

```diff
@@ -218,79 +218,6 @@ public function test_get_notes() {
 		$this->assertEquals( 4, count( $notes ) );
 	}
 
-	/**
-	 * Test getting notes when the user is in tasklist experiment returns notes of size `per_page` without any filters.
-	 *
-	 * @since 3.5.0
-	 */
-	public function test_getting_notes_when_user_is_in_tasklist_experiment_returns_unfiltered_notes() {
```

Comment:
@jaclync Just want to double check that this experiment is no longer being run and that these tests are safe to remove.

It looks like `wc-admin-complete-store-details` may have already been removed and this PR will remove `wc-admin-update-store-details`.

Reply:
sounds good to remove 👍 I searched for the experiment `woocommerce_tasklist_progression_headercard` in ExPlat and all the experiments are disabled, the last one ended on March 22, 2022. I didn't work on the experiment, if you'd like to double check the experiment owner is pmcpinto.